planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/scene.hpp

```diff
@@ -109,14 +109,16 @@ class NormalLaneChange : public LaneChangeBase
 
   bool isStoppedAtRedTrafficLight() const override;
 
-  TurnSignalInfo get_current_turn_signal_info() override;
+  TurnSignalInfo get_current_turn_signal_info() const override;
```

Comment:
```suggestion
  TurnSignalInfo get_current_turn_signal_info() const final;
```
It is `override` previously, but since the modules that inherits `NormalLaneChange` doesn't actually change anything, it is better to change to `final`